Services/UI/OJS.Services.Ui.Models/Contests/ContestServiceModel.cs

```diff
@@ -209,5 +209,6 @@ public bool ResultsArePubliclyVisible
             .ForMember(d => d.UserIsAdminOrLecturerInContest, opt => opt.Ignore())
             .ForMember(d => d.UserCanCompete, opt => opt.Ignore())
             .ForMember(d => d.UserIsParticipant, opt => opt.Ignore())
-            .ForMember(d => d.IsActive, opt => opt.Ignore());
+            .ForMember(d => d.IsActive, opt => opt.Ignore())
+            .ReverseMap();
```

Comment:
What is the benefit of using .ReverseMap() in this particular case?

Reply:
I need the contest for the method IsUserLecturerInContest. The service model does not have the properties that are needed to check if the user is lecturer in the contest. The easiest way to get the contest was just to map the service model to the contest on line 107: 
var contest = await this.contestsDataService.GetByProblemId<ContestServiceModel>(submissionDetailsServiceModel!.Problem.Id).Map<Contest>();